src/http/controllers/dfract.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Body, CacheInterceptor, Controller, Get, Post, Req, UseInterceptors } from '@nestjs/common';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+
+import { AssetService } from '@app/services';
+import { DataResponse, DataResponseMetadata, AssetInfo, AssetHistorical } from '@app/http/responses/';
+import { ExplorerRequest } from '@app/utils';
+import { AssetRequest } from '@app/http/requests';
+
+@ApiTags('dfract')
+@Controller('dfract')
+@UseInterceptors(CacheInterceptor)
+export class DfractController {
+    constructor(private readonly _assetService: AssetService) {}
+
+    @ApiOkResponse({ status: 200, type: AssetInfo })
+    @Get('assets/latest')
+    async getLatestAsset(@Req() request: ExplorerRequest): Promise<DataResponse> {
+        const [result, total] = await this._assetService.fetchLatestMetrics(request.pagination.skip);
+
+        return new DataResponse({
+            result: result,
+            metadata: new DataResponseMetadata({
+                page: request.pagination.page,
+                limit: request.pagination.limit,
+                items_count: result.length,
+                items_total: total,
+            }),
+        });
+    }
+
+    @ApiOkResponse({ status: 200, type: AssetHistorical })
+    // example-1: assets/since {metrics: atom_apy, since: 2022-11-04T10:45:35.919Z}
```

Comment:
There is a decorator designed to handle examples in Swagger, instead of inline comment